# Single_corol_colony_transcriptomes_physiology
Electronic Supplementary Material Table Legends, Figures, and Figure Legends for: Physiological and transcriptomic variability indicative of differences in key functions within a single coral colony

Jeana L. Drake1+, Assaf Malik1+, Yotam Popovits1+, Oshra Yosef1, Eli Shemesh1, Jarosław Stolarski2, Dan Tchernov1, Daniel Sher1,3* and Tali Mass1,3*

1Department of Marine Biology, University of Haifa, 199 Aba Koushy Avenue, Haifa, Israel, 3498838
2Institute of Paleobiology, Polish Academy of Sciences, Twarda 51/55, Warsaw, Poland, 00-818
3Morris Kahn Marine Research Station, The Leon H. Charney School of Marine Sciences, University of Haifa, Sdot Yam, Israel
+These authors contributed equally
*Corresponding authors

Tables
Table S1. Location information for branches used for transcriptional analysis. DESeq2 analysis used the additive model output so that the top-most branch or 9th branch, which points upward and thus does not have a defined cardinal direction, was not included.

Table S2. DE gene count statistics. The counts of over-expressed/under-expressed/unchanged/ NA genes using DESeq2 additive and interaction models. Also included are read count and filtered genes statistics.  Both host Stylophora pistillata and photosymbiont (reads mapped to the Symbiodinium microadriaticum genome, NCBI GCA_001939145.1) data are shown.

Table S3. S. pistillata additive model DE. DESeq2 additive model statistics of DE and gene annotations.

Table S4. S. pistillata interaction model DE. DESeq2 interaction model statistics of DE and gene annotations.

Table S5. Photosymbiont additive model DE. DESeq2 additive model statistics of DE and gene annotations.

Table S6. Dissolved oxygen concentration and total SYBR-stained or chlorophyll auto-fluorescence-based microbial cell counts and measured inside and outside five S. pistillata colonies in situ.

Table S7. Statistical analysis of environmental parameters measured between branches and at colony peripheries, and physiological parameters measured at two tips, a junction, and a base segment on each of five branches distributed throughout the S. pistillata colony.

Table S8. – Host and photosymbiont factor effect Adonis (non-parametric multivariate analysis of variance, known as NPMANOVA or PERMANOVA). The Adonis test, implemented in Vegan, was used to test the effect of factors branch axis, ring height, and cardinal direction, based on NMDS two-dimensional sample clustering.

Table S9. S. pistillata vs . Acropora spp. [1] DE tip vs base statistics for putative orthologous genes found using reciprocal BLAST.

Table S10. S. pistillata KEGG and GO (Trinotate) enrichment based on the DESeq2 additive-model. Enrichment analysis uses the GOSeq Wallenius (Wall) method.

Table S11. S. pistillata KEGG and GO (Trinotate) enrichment based on the DESeq2 interaction-model. Enrichment analysis uses the GOSeq Wallenius (Wall) method.

Table S12. - Photosymbiont GO enrichment (Uniprot database) of DE genes based on the DESeq2 additive-model. Enrichment analysis uses the Gene Set Enrichment Analysis (GSEA) method.

Table S13. Physiological parameters measured along five branches from the same single colony of S. pistillata from which differential gene expression was also examined.

Table S14. DE of putative toxins sequenced in this study obtained by reciprocal blast against known toxins from the literature.

Table S15. DE of orthologous putative toxins between S. pistillata and Acropora spp [1]. Most putative toxins from S. pistillata do not have orthologs in the Acropora spp. dataset.

Table S16. DE of known biomineralization-related genes. 60 genes code for proteins recently sequenced from S. pistillata skeleton [2]. CARPs 1, 2, and 3 [3] and a putative bicarbonate transporter [4] were appended to the list. Not all genes associated with known coral skeletal proteins were sequenced here.

Table S17. DE of orthologous biomineralization-related genes between S. pistillata and Acropora spp [1]. Just under half of such genes from S. pistillata have orthologs in the Acropora spp. dataset.

Supplementary Material Availability
The data underlying this article are available in the article, in its online supplementary material, and on GitHub in the following repository: https://github.com/jeanadrake/Single_corol_colony_transcriptomes_physiology.

Figures
 
Supplementary Figure 1. Percent fluorescence, relative to the injected fluoresceine dye aliquot’s fluorescence, of various volumes of water extracted by syringe either on the opposite side of the branch from the dye injection (a) or at colony peripheries (b). Fluorescence higher than the background water was not observed for extraction volumes of 1.6 ml or smaller.

Supplementary Figure 2. Photosymbiont MA plots. The plots show the genes' mean normalized counts vs. log2fold change, where red dots represent significant DE genes, for North-South and East-South comparisons.  

Supplementary Figure 3. Normalized mV measurements as proxies of DO measured along transects from inside to just outside five coral colonies growing in the same nursery as the colony examined at high spatial resolution for transcriptional, physiological, and morphological differences. Measurements for each colony were standardized to the ‘Periphery’ measurement of each transect for statistical analysis of DO differences by general location within a colony.  

Supplementary Figure 4. Single nucleotide polymorphism (SNP) identity by state. SNP identity by state heatmap showing the level of SNP identity between samples (0-1.0 indicates 0-100% identity), based on the Broad Institute’s GATK.

Supplementary Figure 5. Interaction heatmap of host genes. The heatmap on the left shows the relative expression values of a subset of S. pistillata genes that demonstrate a significant interaction between base-tip (position factor) versus middle-bottom (ring height factor). The heatmap on the right represents the log2FC of the interactions. For clarity, only interactions for tips and bases are shown. 

Supplementary Figure 6. Select measures of photophysiology were not significantly different between branch locations across the single S. pistillata colony measured here, as chlorophyll content (a, c) and photosymbiont abundance (b, d) by either surface area (a, b) or total protein (c, d).

Supplementary Figure 7. Gene expression patterns of putative toxins as log2FC (a, c), and log10FPKM (b, d) between branch tips and junctions (a, b) and junctions and bases (c, d). Only significantly DE toxin genes are shown (p<0.05). An uncharacterized gene, for which reciprocal BLAST hits suggest it is a SCRiP (Supplementary Table 14), is noted with a red star (b).

Supplementary Figure 8. Reciprocal protein blast information (a) and sequence alignment (b) for the D-Pocilopotoxin-Spi1 in (Ben-Ari et al., 2018) (Stylophora_pistillata_9725 from (Bhattacharya et al., 2016)) and the gene27671 (rna35563) annotated here as DELTA-stichotoxin-She4a-like (NCBI accession number XP_022777760.1).

Supplementary Figure 9. Gene expression patterns of known biomineralization-related genes as log2FC (a, c), and log10FPKM (b, d) between branch tips and junctions (a, b) and junctions and bases (c, d). Only significantly DE biomineralization genes are shown (p<0.05).

References
1.	Hemond E.M., Kaluziak S.T., Vollmer S.V. 2014 The genetics of colony form and function in Caribbean Acropora corals. BMC Genomics 15(1), 1133.
2.	Peled Y., Drake J.L., Malik A., Almuly R., Lalzar M., Morgenstern D., Mass T. 2020 Optimization of skeletal protein preparation for LC–MS/MS sequencing yields additional coral skeletal proteins in Stylophora pistillata. BMC Materials 2(1), 8. (doi:10.1186/s42833-020-00014-x).
3.	Mass T., Drake Jeana L., Haramaty L., Kim J.D., Zelzion E., Bhattacharya D., Falkowski Paul G. 2013 Cloning and characterization of four novel coral acid-rich proteins that precipitate carbonates in vitro. Curr Biol 23(12), 1126-1131. (doi:http://dx.doi.org/10.1016/j.cub.2013.05.007).
4.	Zoccola D., Ganot P., Bertucci A., Caminiti-Segonds N., Techer N., Voolstra C.R., Aranda M., Tambutté E., Allemand D., Casey J.R. 2015 Bicarbonate transporters in corals point towards a key step in the evolution of cnidarian calcification. Scientific Reports 5.
5.	Ben-Ari H., Paz M., Sher D. 2018 The chemical armament of reef-building corals: inter-and intra-specific variation and the identification of an unusual actinoporin in Stylophora pistilata. Scientific Reports 8(1), 1-13.
