Supplementary Table 1. Dissolved oxygen concentration and total SYBR-stained or chlorophyll auto-fluorescence-based microbial cell counts and measured inside and outside five S. pistillata colonies in situ.

Supplementary Table 2. Statistical analysis of environmental parameters measured between branches and at colony peripheries, and physiological parameters measured at two tips, a junction, and a base segment on each of five branches distributed throughout the S. pistillata colony.

Supplementary Table 3. Location information for branches used for transcriptional analysis. DESeq2 analysis used the additive model output so that the top-most branch or 9th branch, which points upward and thus does not have a defined cardinal direction, was not included.

Supplementary Table 4. DE gene count statistics. The counts of over-expressed/under-expressed/unchanged/ NA genes using DESeq2 additive and interaction models. Also included are read count and filtered genes statistics.  Both host Stylophora pistillata and photosymbiont (reads mapped to the Symbiodinium microadriaticum genome, NCBI GCA_001939145.1) data are shown.

Supplementary Table 5. S. pistillata additive model DE. DESeq2 additive model statistics of DE and gene annotations.

Supplementary Table 6. S. pistillata interaction model DE. DESeq2 interaction model statistics of DE and gene annotations.

Supplementary Table 7. Photosymbiont additive model DE. DESeq2 additive model statistics of DE and gene annotations.

Supplementary Table 8. Host and photosymbiont factor effect Adonis (non-parametric multivariate analysis of variance, known as NPMANOVA or PERMANOVA). The Adonis test, implemented in Vegan, was used to test the effect of factors branch axis, ring height, and cardinal direction, based on NMDS two-dimensional sample clustering.

Supplementary Table 9. S. pistillata vs . Acropora spp. (Hemond et al.) DE tip vs base statistics for putative orthologous genes found using reciprocal BLAST.

Supplementary Table 10. S. pistillata KEGG and GO (Trinotate) enrichment based on the DESeq2 additive-model. Enrichment analysis uses the GOSeq Wallenius (Wall) method.

Supplementary Table 11. S. pistillata KEGG and GO (Trinotate) enrichment based on the DESeq2 interaction-model. Enrichment analysis uses the GOSeq Wallenius (Wall) method.

Supplementary Table 12. - Photosymbiont GO enrichment (Uniprot database) of DE genes based on the DESeq2 additive-model. Enrichment analysis uses the Gene Set Enrichment Analysis (GSEA) method.

Supplementary Table 13. Physiological parameters measured along five branches from the same single colony of S. pistillata from which differential gene expression was also examined.

Supplementary Table 14. DE of putative toxins sequenced in this study obtained by reciprocal blast against known toxins from the literature.

Supplementary Table 15. DE of orthologous putative toxins between S. pistillata and Acropora spp (Hemond et al.). Most putative toxins from S. pistillata do not have orthologs in the Acropora spp. dataset.

Supplementary Table 16. DE of known biomineralization-related genes. 60 genes code for proteins recently sequenced from S. pistillata skeleton (Peled et al., 2020). CARPs 1, 2, and 3 (Mass et al., 2013) and a putative bicarbonate transporter (Zoccola et al., 2015) were appended to the list. Not all genes associated with known coral skeletal proteins were sequenced here.

Supplementary Table 17. DE of orthologous biomineralization-related genes between S. pistillata and Acropora spp (Hemond et al.). Just under half of such genes from S. pistillata have orthologs in the Acropora spp. dataset.
